modules/beta-autopilot-private-cluster/variables.tf

```diff
@@ -394,12 +394,6 @@ variable "enable_confidential_nodes" {
   default     = false
 }
 
-variable "enable_gcfs" {
```

Comment:
Could we add a note in the upgrade guide about the context of these being removed?

Reply:
Yes - This auto-merged, but I'll open a docs only PR to do that.

Reply:
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/2261